CHANGES.md

```diff
@@ -4,6 +4,7 @@ Change Log
 ### 1.1.1 2017-04-25
 
 * Fixed `CHANGES.md` formatting.
+* Change texture sampling to use NEAREST_MIPMAP_LINEAR by default [#83](https://github.com/AnalyticalGraphicsInc/obj2gltf/pull/83).
```

Comment:
Was 1.1.1 released?  Or should this go in a new `TODO` version until it is deployed?